model/optimality_model.py

```diff
@@ -2,15 +2,13 @@
 from model.optimality_constraints import OptimalityConstraints
 from model.optimality_objective import OptimalityObjective
 from model.optimality_variables import OptimalityVariables
-from utils.weights import get_weights
+
 
 
 class OptimalityModel(BaseModel):
     def __init__(self, name, problem, data):
         super(OptimalityModel, self).__init__(name, problem, data)
 
-        self.weights = get_weights()
```

Comment:
Seems to still be in use on line 36

Reply:
See the same argumentation in the previous review.